spec/namespaces/search_pipeline.yaml

```diff
@@ -4,14 +4,15 @@ info:
   description: OpenSearch Search_pipeline API
   version: 1.0.0
 paths:
-  /_search/pipeline/{pipeline}:
+  /_search/pipeline/{id}:
```

Comment:
@nhtruong  Do you think we should replace path param name from `pipeline` to `id`?

Reply:
Both make sense to me. If the endpoints using this paths haven't made it to any clients then we're free to rename without introducing breaking changes. I know that Ruby and JS clients haven't implemented these yet. @Xtansia what about Java, Net, and Rust?

Reply:
Hasn't been added into any of those clients

Reply:
@saimedhi Then this name change is fine!